assets/css/animation.css

```diff
@@ -272,6 +272,14 @@
   }
 }
 
+.pulse-gentle {
+  animation: pulse-gentle 3s ease-in-out infinite;
+}
+@keyframes pulse-gentle {
+  0%, 100% { transform: scale(1); }
+  50% { transform: scale(1.02); }
+}
+
```

Comment:
3 seconds is too long. People will browse not as long. Do some basic market research for how long people will spend on this part of the page and adjust your approach. You also can be creative about what animation you use.